docs/execution-providers/OpenVINO-ExecutionProvider.md

```diff
@@ -80,23 +84,41 @@ Refer to [Configuration Options](#configuration-options) for more information ab
 
 This flow enables users to import/load the pre-compiled blob directly if available readily. This option is enabled by explicitly setting the path to the blob using environment variables and setting the OV_USE_COMPILED_NETWORK flag to true.
 
+This flow only works for MyriadX(VPU) device.
+
 For Linux:
 ```
 export OV_USE_COMPILED_NETWORK=1
-export OV_BLOB_PATH =<path to the blob>
+export OV_BLOB_PATH=<path to the blob>
+Example: export OV_BLOB_PATH=/home/blobs_dir/model.blob
 ```
 
 For Windows:
 ```
 set OV_USE_COMPILED_NETWORK=1
-set OV_BLOB_PATH =<path to the blob>
+set OV_BLOB_PATH=<path to the blob>
+Example: set OV_BLOB_PATH=\home\blobs_dir\model.blob
 ```
 
+compile_tool:
+
+The device specific Myriadx blobs can be generated using an offline tool called compile_tool from OpenVINO Toolkit.[documentation](https://docs.openvinotoolkit.org/latest/openvino_inference_engine_tools_compile_tool_README.html).
+
 ### Support for INT8 Quantized models
 
 Starting from the OpenVINO EP 2021.4 Release, int8 models will be supported on CPU and GPU.
 However, int8 support won't be available for VPU.
 
+### Support for Weights saved in external files
+
+Starting from the OpenVINO EP 2021.4 Release, support for external weights is added. OpenVINO™ EP now  supports ONNX models that store weights in external files. It is especially useful for models larger than 2GB because of protobuf limitations.[documentation](https://docs.openvinotoolkit.org/latest/openvino_docs_IE_DG_ONNX_Support.html).
+
+Converting and Saving an ONNX Model to External Data:
+Use the ONNX API's.[documentation](https://github.com/onnx/onnx/blob/master/docs/ExternalData.md#converting-and-saving-an-onnx-model-to-external-data).
+
+Note: Install the latest ONNX Python package using pip to run these ONNX Python API's successfully.
+
```

Comment:
Add a note about saving external weights filename as data/weights, or not moving the weights after conversion. 

Or could also provide example 
`# Load onnx model`
`onnx.save_model(onnx_model, 'path/to/save/the/model.onnx', save_as_external_data=True, all_tensors_to_one_file=True, location='data/weights', size_threshold=1024, convert_attribute=False)`
`# Run with OpenVINO EP`